src/moondawg/moondawg/diagnostics.py

```diff
@@ -15,7 +12,7 @@ def __init__(self):
 
         # create heartbeat
```

Comment:
explain what this means and does

Reply:
Diagnostics was kinda neglected in this PR, a lot can be improved for when you and Tony tackle it for improvement